src/api/auth.ts

```diff
@@ -407,7 +407,8 @@ export function publishPackage(
                 owners: [ username ],
                 organization: organization ? organization : '',
                 teamPermissions: permissions,
-                memberPermissions: [{ name: username, permission: 'read-write' }]
+                memberPermissions: [{ name: username, permission: 'read-write' }],
```

Comment:
@Izak88 consider the use case when admin wants to temporary restrict the access to some packages or hide them from certain users, then you just i.e. set `read: false`. In case of Jan's format you don't need to parse anything.